README.md

```diff
@@ -1,4 +1,4 @@
 # Brute Force Protection App
-[![Build Status](https://travis-ci.org/owncloud/brute_force_protection.svg?branch=master)](https://travis-ci.org/owncloud/brute_force_protection)
+[![Build Status](https://drone.owncloud.com/api/badges/owncloud/brute_force_protection/status.svg)](https://drone.owncloud.com/owncloud/brute_force_protection)
 

```

Comment:
Please add codecov badge as well - THX a lot!

Reply:
@DeepDiver1975, I have made the changes, please review it again.